i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/core/fragments/_basic-token-requirements.mdx

```diff
@@ -0,0 +1,11 @@
+你需要一个满足如下条件的 Access Token：
+
+<ul>
+  <li>是一个有效的 token。</li>
+  <li>颁发自你的 Logto 实例。</li>
+  <li>
+    它的 audience （<code>aud</code>，即 API indentifier）指向 <code>https://api.logto.io</code>。
```

Comment:
```suggestion
    它的 audience（<code>aud</code>，即 API indentifier）指向 <code>https://api.logto.io</code>。
```
no need to add a whitespace between the English word and the full-width character.